cmd/cloud_login.go

```diff
@@ -115,6 +134,26 @@ func (c *cmdCloudLogin) run(cmd *cobra.Command, _ []string) error {
 		newCloudConf.Token = null.StringFrom(vals["Token"])
 	}
 
+	if newCloudConf.Token.Valid {
+		client := cloudapi.NewClient(
+			c.globalState.Logger,
+			newCloudConf.Token.String,
+			consolidatedCurrentConfig.Host.String,
+			consts.Version,
+			consolidatedCurrentConfig.Timeout.TimeDuration(),
+		)
+
+		var res *cloudapi.ValidateTokenResponse
+		res, err = client.ValidateToken()
+		if err != nil {
+			return err
```

Comment:
We do some raw error returns inside the `client.ValidateToken()`. Let's wrap error here to make it more human-friendly, like:
```suggestion
			return fmt.Errorf("can't validate the API token: %s", err)
```

or a better version